bundles/org.openhab.core.automation.module.script/src/main/java/org/openhab/core/automation/module/script/profile/ScriptProfile.java

```diff
@@ -43,6 +43,8 @@ public class ScriptProfile implements StateProfile {
 
     public static final String CONFIG_TO_ITEM_SCRIPT = "toItemScript";
     public static final String CONFIG_TO_HANDLER_SCRIPT = "toHandlerScript";
+    public static final String CONFIG_TO_HANDLER_COMMAND_SCRIPT = "toHandlerCommandScript";
```

Comment:
If we change the naming anyway: Wouldn't `stateFromItemScript` and `commandFromItemScript` be more clear? I'm not sure here, it just came to my mind.

Reply:
Agreed! Having `toItem` and `fromItem` IMO is better to understand for the user as `toHandler` and `toItem`, because I am not sure whether most users actually know what the/a (Thing) handler is.